hypha/settings/salesforce.py.example

```diff
@@ -0,0 +1,14 @@
+SALESFORCE_DB = {
+    'ENGINE': 'salesforce.backend',
+    'CONSUMER_KEY': 'Consumer Key from Salesforce App',
+    'CONSUMER_SECRET': 'Consumer Secret from Salesforce APP',
+    'USER': 'user@email.com',
+    'PASSWORD': 'Password and security token concatenated',
+    'HOST': 'https://login.salesforce.com' # Or test or URL of salesforce org
+}
```

Comment:
I think we should make all these environment variables. On hosting environments like Heroku you can't upload a separate configuration file but setting environment variables is easy and reasonable secure.

Reply:
Having the option to use environment variables or a settings file might be the absolut best.